cmake/chimeraFunctions.cmake

```diff
@@ -135,7 +135,7 @@ function(add_chimera_binding)
     ExternalProject_Add("${binding_TARGET}_REBUILD"
         DOWNLOAD_COMMAND ""
         INSTALL_COMMAND ""
-        BUILD_COMMAND make "${binding_TARGET}_SOURCES"
+        BUILD_COMMAND $(MAKE) "${binding_TARGET}_SOURCES"
```

Comment:
😱 I've never seen this feature before. Nice find.

Can you please add the StackOverflow link as a comment in the source code? I am concerned that we someone - possibly myself - will be baffled by this in the future.